pkg/v3/plugin/plugin.go

```diff
@@ -38,7 +39,10 @@ func New(
 		PrebuildHooks: []func(ocr2keepersv3.AutomationOutcome) error{
 			ltFlow.ProcessOutcome,
 		},
-		ResultSource: rs,
+		ResultSource:       rs,
+		ReportEncoder:      encoder,
+		ReportGasLimit:     5_000_000,
+		MaxUpkeepBatchSize: 1,
```

Comment:
Am i understanding this right?

We have 2 `outcome.Performable` as a batch and the 2 lives in a report, as indicated in L96 of ocr3.go.

Reply:
Yes. These are just hard-coded values for now, but will be set by configurations going forward.